apiserver/src/middlewares/authenticate.ts

```diff
@@ -1,9 +1,10 @@
-import { verifyToken, signTokens } from "../helpers/jwt";
-import { setCookies } from "../helpers/cookies";
-import { Request, Response } from "express";
-
-const accessSecret = process.env.ACCESS_SECRET || "abcdef";
-const refreshSecret = process.env.REFRESH_SECRET || "ghijkl";
+import moment from "moment";
```

Comment:
moment.js dropped support on the library. 
Maybe use another one like date-fns or other...

Reply:
Nice catch. Suggest Luxon. Changed.